src/pages/Cores/LearnMore/ArticlesData/index.ts

```diff
@@ -0,0 +1,159 @@
+import {Article} from 'types';
+import Benefits from 'assets/articles/benefit.png';
+import MissionImg from 'assets/articles/m_statement.png';
+import Tag from 'assets/articles/tag_icon.png';
+import UseCases from 'assets/articles/UseCases.png';
+import VisionImg from 'assets/articles/v_statement.png';
+import SystemArc from 'assets/articles/Archi.png';
+
+const data = {
+  amount: 5,
+  id: 'dc348eac-fc89-4b4e-96de-4a988e0b94e1',
+  payload: {
+    message: 'Hey',
+    recipient: `995bd2a4db610062f404510617e83126fa37e28368
+                05975f334108b55523634c`,
+    sender: `eb01f474a637e402b44407f3c1044a0c4b59261515d50
+              be9abd4ee34fcb9075b`,
+    signature: `4b86d923cfc7603f39e1d0c36afcec2e454c624b1d
+                c4dd03bf6764e4662162644d0a78c864d16bb7e4608a76db6df0e84
+                2a550c52d4811f81d8049f273da8a01`,
+    transaction_fee: 1,
+  },
+};
+
+const article_detail = `The system operates as a decentralized network of networks, allowing individuals to create and distribute their own digital currency known as "coins." Each core server in the network maintains a separate ledger. The minting and distribution of new coins are determined by the server owner. \n
+      \n
+      Coins within the network serve various purposes. They enable secure messaging between devices and can also be transferred as a form of value between users. The network architecture permits devices to store data locally, removing the need for a centralized database and facilitating the development of distributed applications. \n
+      \n
+      To enable messaging and value transfer, a standardized protocol is employed. Although core servers are not directly connected, clients can connect to multiple cores, effectively linking them through client devices.\n
+      \n
+      Notably, messaging and value transfers take place only within the same core server. Each server maintains its independent ledger, ensuring that each users transaction history remains within a single server as the definitive record of truth.`;
+
+export const articlesData: Article[] = [
```

Comment:
instead of keeping everything in one list, I think we can also create separate constants for each Learn More Card Content.
it would simplify things

Reply:
ps: I'm not sure how we are rendering these numbers right now.
<img width="233" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/341e54e6-1c41-4709-bb57-9dceb134e718">

but can we simply have a flag for this: `showNumbering`? if true, then these list numbers are rendered, else not.
Or something else similar?